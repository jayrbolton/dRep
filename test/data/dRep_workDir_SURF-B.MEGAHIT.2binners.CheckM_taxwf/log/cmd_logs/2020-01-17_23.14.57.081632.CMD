nucmer --mum -p /kb/module/work/tmp/dRep_workDir_d7f80abb-1c52-43db-9fae-de726bb861a2/data/ANImf_files/34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta/34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta_vs_34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta -c 65 -g 90 /kb/module/work/tmp/binsPooled_d7f80abb-1c52-43db-9fae-de726bb861a2/34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta /kb/module/work/tmp/binsPooled_d7f80abb-1c52-43db-9fae-de726bb861a2/34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta; delta-filter -r -q /kb/module/work/tmp/dRep_workDir_d7f80abb-1c52-43db-9fae-de726bb861a2/data/ANImf_files/34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta/34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta_vs_34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta.delta > /kb/module/work/tmp/dRep_workDir_d7f80abb-1c52-43db-9fae-de726bb861a2/data/ANImf_files/34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta/34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta_vs_34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta.delta.filtered
