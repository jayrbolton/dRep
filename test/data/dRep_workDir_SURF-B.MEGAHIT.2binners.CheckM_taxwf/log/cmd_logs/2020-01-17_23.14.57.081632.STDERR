1: PREPARING DATA
2,3: RUNNING mummer AND CREATING CLUSTERS
# reading input file "/kb/module/work/tmp/dRep_workDir_d7f80abb-1c52-43db-9fae-de726bb861a2/data/ANImf_files/34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta/34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta_vs_34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta.ntref" of length 4051621
# construct suffix tree for sequence of length 4051621
# (maximum reference length is 536870908)
# (maximum query length is 4294967295)
# process 40516 characters per dot
#....................................................................................................
# CONSTRUCTIONTIME /usr/local/bin/MUMmer3.23/mummer /kb/module/work/tmp/dRep_workDir_d7f80abb-1c52-43db-9fae-de726bb861a2/data/ANImf_files/34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta/34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta_vs_34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta.ntref 1.70
# reading input file "/kb/module/work/tmp/binsPooled_d7f80abb-1c52-43db-9fae-de726bb861a2/34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta" of length 4051620
# matching query-file "/kb/module/work/tmp/binsPooled_d7f80abb-1c52-43db-9fae-de726bb861a2/34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta"
# against subject-file "/kb/module/work/tmp/dRep_workDir_d7f80abb-1c52-43db-9fae-de726bb861a2/data/ANImf_files/34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta/34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta_vs_34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta.ntref"
# COMPLETETIME /usr/local/bin/MUMmer3.23/mummer /kb/module/work/tmp/dRep_workDir_d7f80abb-1c52-43db-9fae-de726bb861a2/data/ANImf_files/34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta/34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta_vs_34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta.ntref 5.29
# SPACE /usr/local/bin/MUMmer3.23/mummer /kb/module/work/tmp/dRep_workDir_d7f80abb-1c52-43db-9fae-de726bb861a2/data/ANImf_files/34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta/34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta_vs_34837-2-1__SURF-B.MEGAHIT.metabat.CheckM__Bin.012.fasta.ntref 7.79
4: FINISHING DATA
